js/binance.js

```diff
@@ -6043,6 +6045,59 @@ module.exports = class binance extends Exchange {
         };
     }
 
+    async repayMarginLoan (code, amount, symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+        }
+        const currency = this.currency (code);
+        const request = {
+            'asset': currency['id'],
+            'amount': amount,
+        };
+        const defaultMargin = this.safeString (params, 'marginMode', 'cross'); // cross or isolated
+        params = this.omit (params, 'marginMode');
+        const marginMode = this.safeString2 (this.options, 'defaultMarginMode', 'marginMode', defaultMargin);
```

Comment:
The `params`-overrides must take precedence over options, so if we specify `marginMode` in params and in options, the one in params has priority.